src/platform/elements/header/Header.ts

```diff
@@ -55,20 +55,23 @@ export class NovoUtilActionComponent {
         <ng-content></ng-content>
     `,
 })
-export class NovoHeaderComponent implements OnInit {
+export class NovoHeaderComponent implements OnInit, OnChanges {
     @Input() public title: string;
     @Input() public subTitle: string;
     @HostBinding('class') public headerClass: string = 'novo-header';
     @HostBinding('attr.theme')
-    @Input()
-    public theme: string;
+    @Input() public theme: string;
     @Input() public icon: string;
     @HostBinding('class.condensed') @Input() public condensed: boolean = false;
 
     public inverse: string = 'inverse';
     public iconClass: string;
 
     public ngOnInit(): void {
+        this.ngOnChanges();
+    }
+
+    public ngOnChanges(): void {
```

Comment:
Actually can you just remove ngOnInit and onChanges? Use setters/getters instead. Look at button for what it  looks like

Reply:
Done